data/hai/hai news.txt

```diff
@@ -165,3 +165,139 @@ news "reformed pirate in Hai space"
 			"The Hai will pay you to rush cargo to human space and back. Don't die trying."
 		word
 			`"`
+
+news "unfettered hai dialog"
+	location
+		government "Hai (Unfettered)"
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Unfettered warlord"
+			"Hai youth"
+			"Hai warlord"
+	message
+		word
+			`"`
+		word
+			"How did you get here? You should've been blasted to bits by our ships."
+			"I came here because this is the only place in the galaxy where I feel wanted."
+			"Weak human. Why do you dare to come to the land of the true Hai?"
+			"Fettered Hai are soft and weak-willed. They care not for true struggle or challenge."
+			"You call me 'cute', and I'll run you through with a machete."
+			"I seek challenge, and what greater challenge is there than to oppose the Drak, those with the power of gods?"
+			"Sometimes I miss my family and friends back home, but then I remember that here I am a respected warrior. Back there I was a slave to the Drak."
+			"Our ships can beat anything you stupid monkeys can make."
+			"This sure beats being a factory worker."
+			"Without change, a civilization will decay and fall. This will happen to the fettered Hai, as it has happened to countless empires throughout history."
+			"What are you looking at, monkey?"
+			"I can take on a human like you any day."
+			"You looking for a fight, monkey?"
+			"Last week we plundered a human ship that was stupid enough to come up here. Who knows, maybe you'll be next."
+			"We just plundered a merchant convoy that was loaded with food. We'll feast well tonight."
+		word
+			`"`
+news "Unfettered Regret"
+	name
```

Comment:
You're adding six news types to Unfettered world, two of which are Hai showing regret, so a third of the time the news you get would be about Unfettered regretting their actions, which seems like a dangerously low morale. We could put random conditions in the to shows of these news types to reduce the odds of seeing an Unfettered expressing regret.
```suggestion
	to show
		random < 10
	name
```